templates/takeovers/_german_takeover_k8.html

```diff
@@ -1,4 +1,4 @@
-<div data-lang="{{ locale }}" lang="de" class="p-strip--image is-dark js-takeover {% if not default %}u-hide{% endif %}" style="background-color:#326DE6">
```

Comment:
@nottrobin thank you for taking the time out. I noticed you removed the lang attribute from the takeover. I believe it is used to generate to filter for the takeovers. The idea of adding it was to be able to add English Language takeovers to German sites for example.

Reply:
It's not actually used by JavaScript, only the `data-lang` is. But maybe we need the `lang` attribute too, I'll continue that discussion on ant's comment: https://github.com/canonical-websites/www.ubuntu.com/pull/4371#discussion_r232564652